agg.sh

```diff
@@ -12,8 +12,8 @@ fi
 
 while true ; do
         date |tee -a $LOG_FILE
-        python -m generate_functional_agg |tee -a $LOG_FILE
-        python -m generate_metap_agg 2>&1 |tee -a $LOG_FILE
+        python -m generate_metag_metaat_functional_agg |tee -a $LOG_FILE
```

Comment:
I think we'll want the "2>&1" in this line too so we record the error messages properly.  Though I am not positive.